iontestdata_1_1/good/equivs/macros/invoked_in_a_list.ion

```diff
@@ -0,0 +1,6 @@
+$ion_1_1
+(
+  [a, "abc"]
+  [(:values a "abc")]
+  [a, (:values "abc")]
+)
```

Comment:
Is the idea that more kinds of macro invocations will be added to these `*_in_a_*` files, or should these be named e.g. `values_invoked_in_a_list.ion`?

Reply:
The idea was that there would be some tests that specifically focused on _where_ the macro invocation occurs, and others that focused on specific macro invocations. Right now, all of our testing frameworks (that I'm aware of) treat each file as one test case when reporting the test results, so I was trying to separate out some location tests from the test that were specific to each macro. Unfortunately, it's not that simple because you can't test invoking a macro in different locations without also testing a specific macro.

As I've thought about it more, I think I'd like to try to rework the format for our test case (at least for Ion 1.1), as long as it's not too much additional work.